### ISSUE TITLE
Accessibility: Carousel controls need labels

### ISSUE THREAD

**Issue by VanessaVenti** [projectblacklight/spotlight#3529]
### Issue Description - WCAG standard
Controls on the carousel slides. The next and previous need aria-labels. 

### Method of discovery
Siteimprove, Wave, keyboard navigation, VoiceOver etc.

### Steps to Reproduce
Add a carousel widget, test using wave or other tool. 
<img width="250" alt="Image" src="https://github.com/user-attachments/assets/48f95759-8020-4c02-bf57-f04196ab317f" />

### Suggested solution
add aria-labels [here?](https://github.com/projectblacklight/spotlight/blob/main/app/views/spotlight/sir_trevor/blocks/_solr_documents_carousel_block.html.erb)

**Comment by sarah-cul** [comment]
When a carousel only has 1 item, screen readers may not like that it provides the description for next and previous buttons because the descriptions are all the same item. Its not technically wrong. 
I wasn't sure how important that was to address because a carousel with just a single item seems like a bad design choice to begin with.  Any thoughts?

**Comment by VanessaVenti** [comment]
Agreed, a carousel with one item seems like an unlikely scenario / bad design choice so I don't think we need to do anything special to account for the at scenario.
